src/platformPackager.ts

```diff
@@ -306,54 +388,63 @@ export abstract class PlatformPackager<DC extends PlatformSpecificBuildOptions>
     })
   }
 
-  private expandPattern(pattern: string, arch: Arch): string {
-    return pattern
-      .replace(/\$\{arch}/g, Arch[arch])
-      .replace(/\$\{os}/g, this.platform.buildConfigurationKey)
-      .replace(/\$\{\/\*}/g, "{,/**/*,/**/.*}")
-  }
-
-  private doCopyExtraFiles(isResources: boolean, appOutDir: string, patterns: Array<Minimatch> | null): Promise<any> {
-    const base = isResources ? this.getResourcesDir(appOutDir) : (this.platform === Platform.MAC ? path.join(appOutDir, `${this.appInfo.productFilename}.app`, "Contents") : appOutDir)
+  private doCopyExtraFiles(isResources: boolean, patterns: Array<FileMatcher> | n): Promise<any> {
```

Comment:
`isResources` is unused. Consider to use IntelliJ IDEA/WebStorm :)
<img width="965" alt="screen shot 2016-08-11 at 07 18 33" src="https://cloud.githubusercontent.com/assets/350686/17579291/d8151324-5f93-11e6-838e-0676dd62e6a6.png">